partner_location_auto_create/views/res_partner_view.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <record id="view_partner_form" model="ir.ui.view" >
```

Comment:
<odoo> replace for all openerp tag reference

Reply:
:+1: